src/components/edit_area/edit_area.tsx

```diff
@@ -108,11 +106,15 @@ export const EditArea = React.forwardRef<EditAreaState, EditAreaProps>((props, r
     onActionResizeStart,
```

Comment:
What are some different extreme cases to test?

- moving an action 2 tracks where it has to go through an action 1 track above it?
- moving really small actions?
- moving actions near the end time? start time?
- moving selected action vs unselected action?
- moving voiceover action (shouldn't be allowed)
- moving avatar action beyond the end of voiceover action? (shouldn't be allowed)
- undo/redo still works
- dragging an action to another track and back to same track in the same motion?